src/ontology/mondo.Makefile

```diff
@@ -264,7 +264,35 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# address-medgen-conflicts-aug2023 pipeline
+# - https://github.com/monarch-initiative/mondo-ingest/issues/273
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+tmp/bad-medgen-xrefs.txt: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/bad_medgen_xrefs_collate.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
+
+tmp/bad-medgen-xrefs-grep-command.sh: tmp/bad-medgen-xrefs.txt
+	python ../scripts/bad_medgen_xrefs_grep_command.py -i tmp/bad-medgen-xrefs.txt -t mondo-edit.obo -o $@
+
+mondo-edit.obo.tmp: tmp/bad-medgen-xrefs-grep-command.sh
+	sh $<
+
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	-diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/bad_medgen_xrefs_update_analyze_diff.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+.PHONY: address-medgen-conflicts-aug2023
+address-medgen-conflicts-aug2023: tmp/report-qc-medgen-conflicts-update-diff.tsv
+	grep -v 'source="UMLS:CN' mondo-edit.obo.tmp > mondo-edit.obo.tmp2.tmp
+	grep -v MEDGEN: mondo-edit.obo.tmp2.tmp > mondo-edit.obo.tmp3.tmp
```

Comment:
Added a 3rd `grep -v`
@matentzn This is the removal of all lines that have the text `MEDGEN:`, as you requested at our meeting today.
You recommended that this be done manually, but this was so easy/fast to do programmatically that I preferred to do it that way. I think it's much better doing it programmatically as well, just in case there are any merge conflicts or something else that would necessitate that I re-run this. 

There were about 30 removals.

<details><summary>Diff</summary>
<p>

```diff
2196d2195
< xref: MEDGEN:546803 {source="UMLS:C0275757"}
2213d2211
< xref: MEDGEN:537148 {source="UMLS:C0259737"}
2346d2343
< xref: MEDGEN:540779 {source="UMLS:C0267556"}
2992d2988
< xref: MEDGEN:575966 {source="UMLS:C0343965"}
3485d3480
< xref: MEDGEN:547234 {source="UMLS:C0276286"}
3533d3527
< xref: MEDGEN:547314 {source="UMLS:C0276386"}
13413d13406
< xref: MEDGEN:508459 {source="UMLS:C0149948"}
16128d16120
< xref: MEDGEN:11567 {source="UMLS:C0038041"}
26833d26824
< xref: MEDGEN:509829 {source="UMLS:C0155174"}
69691d69681
< xref: MEDGEN:208928 {source="UMLS:C0854912"}
151847,151848d151836
< xref: MEDGEN:358137 {source="UMLS:C1868139"}
< xref: MEDGEN:881357 {source="UMLS:C4054549"}
172389d172376
< xref: MEDGEN:82812 {source="UMLS:C0268501"}
350178d350164
< synonym: "XYLT1-CDG - xylosyltransferase 1 congenital disorder of glycosylation" EXACT [MEDGEN:C4750849]
371444d371429
< xref: MEDGEN:231044 {source="UMLS:C1332210"}
404531d404515
< xref: MEDGEN:508876 {source="UMLS:C0152081"}
404569d404552
< xref: MEDGEN:508879 {source="UMLS:C0152087"}
404588d404570
< xref: MEDGEN:57754 {source="UMLS:C0152222"}
404625d404606
< xref: MEDGEN:102349 {source="UMLS:C0156307"}
404825d404805
< xref: MEDGEN:60214 {source="UMLS:C0206708"}
408322d408301
< xref: MEDGEN:447610
409362d409340
< xref: MEDGEN:575892 {source="UMLS:C0343875"}
433950d433927
< def: "Deficient blood distribution to the limbs caused by narrowing or obstruction of the lumen of the peripheral arteries." [MEDGEN:115991]
434966d434942
< xref: MEDGEN:419036 {source="UMLS:C2931284"}
434999d434974
< xref: MEDGEN:75782
435090d435064
< xref: MEDGEN:452941 {source="UMLS:C1291560"}
435118d435091
< xref: MEDGEN:6300 {source="UMLS:C0025306"}
435162d435134
< xref: MEDGEN:418965 {source="UMLS:C2931030"}
435219d435190
< xref: MEDGEN:224821 {source="UMLS:C1279186"}
441678d441648
< xref: MEDGEN:233758 {source="UMLS:C1334968"}

```

</p>
</details>